components/brave_ads/common/features.cc

```diff
@@ -16,6 +16,9 @@ const base::Feature kAdNotifications{"AdNotifications",
 const base::Feature kCustomAdNotifications{"CustomAdNotifications",
                                            base::FEATURE_DISABLED_BY_DEFAULT};
 
+const base::Feature kRequestAdsEnabledApi{"RequestAdsEnabledApi",
```

Comment:
I think this should be disabled by default for now? @mrose17

Reply:
for the moment, disabled by default is good. once we are confident in it, then the API should be available by default

Reply:
Changed feature state to disabled by default.